package.json

```diff
@@ -16,8 +16,8 @@
   },
   "homepage": "https://github.com/amaabca/sensitive-param-filter",
   "devDependencies": {
-    "eslint": "^7.27.0",
-    "jest": "^27.0.4"
+    "eslint": "7.32.0",
+    "jest": "27.1.1"
```

Comment:
Are these what are depending on [path-parse](https://github.com/amaabca/sensitive-param-filter/security/dependabot/yarn.lock/path-parse/open) and [glob-parent](https://github.com/amaabca/sensitive-param-filter/security/dependabot/yarn.lock/glob-parent/open)?

Reply:
Yep